dashboard.json

```diff
@@ -36,6 +36,11 @@
         "title": "News",
         "icon": "radio",
         "items": [
+            {
+                "title": "Pulumi 0.16.0 now available",
```

Comment:
Updating this file is no longer necessary (though, there's no harm in doing so). I'm planning to delete this file in the near future, along with the instructions on updating it.